ebmbot/webserver/github.py

```diff
@@ -62,8 +62,9 @@ def should_deploy(request):
     return data["action"] == "closed" and data["pull_request"]["merged"]
 
 
-def schedule_deploy():
-    """Schedule a deploy of openprescribing."""
+def schedule_deploy(project):
+    """Schedule a deploy of the given project."""
 
+    job = f"{project}_deploy"
     logger.info("Scheduling deploy")
-    scheduler.schedule_job("op_deploy", {}, "#general", "", delay_seconds=60)
+    scheduler.schedule_job(job, {}, "#general", "", delay_seconds=60)
```

Comment:
What error will a client get for a non-matching project? Will it be obvious what the problem is?

Reply:
There's no error before we push a job into the queue, which doesn't seem ideal!

I took this route to avoid coupling the webhook consumer to the config, but that's going to end up letting unknown jobs into the queue and make it trickier to track down problems.

I'll add a check to the webhook so we can catch it before scheduling.